rdr_service/resource/schemas/participant.py

```diff
@@ -219,10 +222,11 @@ class Meta:
         pii_filter = {}  # dict(field: lambda function).
 
 
-class EHRReceiptSchema(Schema):
+class EhrReceiptSchema(Schema):
```

Comment:
FYI:  changed this because originally I had some code that was aided by having the Resource schema class name align with the BQ schema class name (aside from the BQ prefix) and the BQ schema uses this case convention.